sdk/identity/azure-identity/APP_REGISTRATION.md

```diff
@@ -0,0 +1,138 @@
+   | **Accounts in any organizational directory and personal Microsoft accounts** | Select this option to target the widest set of customers.<br><br>By selecting this option, you're registering a *multitenant* application that can also support users who have personal *Microsoft accounts*. Personal Microsoft accounts include Skype, Xbox, Live, and Hotmail accounts. |
+   | **Personal Microsoft accounts** | Select this option if you're building an application only for users who have personal Microsoft accounts. Personal Microsoft accounts include Skype, Xbox, Live, and Hotmail accounts. |
+
+1. Leave **Redirect URI (optional)** alone for now as you configure a redirect URI in the next section.
+1. Select **Register** to complete the initial app registration.
+
+![content](images/portal-02-app-reg-01.png)
+
+When registration finishes, the Microsoft Entra admin center displays the app registration's **Overview** pane. You see the **Application (client) ID**. Also called the *client ID*, this value uniquely identifies your application in the Microsoft identity platform.
+
+> New app registrations are hidden to users by default. When you are ready for users to see the app on their [My Apps page](https://support.microsoft.com/account-billing/sign-in-and-start-apps-from-the-my-apps-portal-2f3b1bae-0e5a-4a86-a33e-876fbd2a4510) you can enable it. To enable the app, in the Microsoft Entra admin center navigate to **Identity** > **Applications** > **Enterprise applications** and select the app. Then on the **Properties** page toggle **Visible to users?** to Yes.
+
+Your application's code, or more typically an authentication library used in your application, also uses the client ID. The ID is used as part of validating the security tokens it receives from the identity platform.
+
+![content](images/portal-03-app-reg-02.png)
+
+# Assign Azure roles using the Azure portal
+
+## Step 1: Identify the needed scope
+
+When you assign roles, you must specify a scope. Scope is the set of resources the access applies to. In Azure, you can specify a scope at four levels from broad to narrow: [management group](https://learn.microsoft.com/azure/governance/management-groups/overview), subscription, [resource group](https://learn.microsoft.com/azure/azure-resource-manager/management/overview#resource-groups), and resource.
+
+![Diagram that shows the scope levels for Azure RBAC.](images/scope-levels.png)
+
+- Sign in to the [Azure portal](https://portal.azure.com).
+
+- In the Search box at the top, search for the scope you want to grant access to. For example, search for **Management groups**, **Subscriptions**, **Resource groups**, or a specific resource.
+
+- Click the specific resource for that scope.
```

Comment:
As a rule of thumb, use "select" instead of "click". You'll want to apply this suggestion elsewhere in the doc.
```suggestion
- Select the specific resource for that scope.
```